pesacheck_meedan_bridge/py/main.py

```diff
@@ -0,0 +1,125 @@
+import json
+import re
+
+import requests
+import sentry_sdk
+import settings
+from check_api import create_mutation_query
+from database import PesacheckDatabase, PesacheckFeed
+
+
+def remove_html_tags(input_string):
+    return re.sub(r"<[^>]*>", "", input_string)
+
+
+def fetch_from_pesacheck():
+    url = "https://api.rss2json.com/v1/api.json"
+    feed_url = settings.PESACHECK_URL
+    params = {
+        "rss_url": feed_url,
+        "api_key": settings.PESACHECK_RSS2JSON_API_KEY,
+        "count": 10,
+        "order_by": "pubDate",
+    }
+    response = requests.get(url, params=params, timeout=60)
+    if response.status_code == 200:
+        return response.json().get("items") or []
+    raise Exception("An Error Occurred fetching data from pesacheck")
+
+
+def post_to_check(query):
+    headers = {
+        "Content-Type": "application/json",
+        "X-Check-Token": settings.PESACHECK_CHECK_TOKEN,
+        "X-Check-Team": settings.PESACHECK_CHECK_WORKSPACE_SLUG,
+    }
+    body = dict(query=query)
+    url = settings.PESACHECK_CHECK_URL
+    response = requests.post(url, headers=headers, json=body, timeout=60)
+    if response.status_code == 200:
+        return response.json()
+    raise Exception(response.text)
+
+
+def store_in_database(feed):
+    db = PesacheckDatabase()
+    db.insert_pesacheck_feed(feed)
+    return feed
+
+
+def post_to_check_and_update(feed):
+    categories = json.loads(feed.categories)
+    language = "fr" if "french" in categories else "en"
+    input_data = {
+        "media_type": "Blank",
+        "channel": 1,
+        "set_tags": categories,
+        "set_status": "verified",
+        "set_claim_description": remove_html_tags(feed.description),
+        "title": remove_html_tags(feed.title),
+        "summary": remove_html_tags(feed.description),
+        "url": feed.link,
+        "language": language,
+        "publish_report": True,
+    }
+    query = create_mutation_query(**input_data)
+    res = post_to_check(query)
```

Comment:
Any reason we separated these? Why not just expose a single post method from check_api?